### PR TITLE
feat: Improve cell type info UX

### PR DESCRIPTION
**Pull request by seve**
https://github.com/chanzuckerberg/single-cell-data-portal/issues/3545

Implements a panel that displays info about a selcted cell type.

### CODE REVIEW

File: frontend/src/views/WheresMyGene/components/Filters/index.tsx [hunk 1]
```diff
@@ -177,75 +174,77 @@ export default memo(function Filters({ isLoading }: Props): JSX.Element {
...
+      <Organism isLoading={isLoading} />
+
+      <Sort areFiltersDisabled={areFiltersDisabled} />
+    </Wrapper>
+    /* </TooltipWrapper> */
   );
 
-  function TooltipWrapper({ children }: { children: ReactElement }) {
-    const Wrapper = areFiltersDisabled ? Tooltip : Fragment;
+  // (seve): temp disable as it was adding a title prop to fragment, leading to errors
```
Comment (seve) [inline frontend/src/views/WheresMyGene/components/Filters/index.tsx:237]: @tihuan

File: frontend/src/common/queries/wheresMyGene.ts [hunk 2]
```diff
@@ -769,37 +762,62 @@ export interface MarkerGenesByCellType {
...
+
+    result = generateTermsByKey(genes, "id");
+
+    return result;
+  }, [data]);
+
+  return useQuery(
+    [USE_MARKER_GENES, cellTypeID, test],
+    // Getting some weird issues with fetching some times (right side bar is not populated),
+    //  this may be related to receiving NaN's from the backend. Going to ignore for now
```
Comment (atarashansky) [inline frontend/src/common/queries/wheresMyGene.ts:796]: Is this issue still relevant / reproducible?

When right sidebar isn't being populated, is it consistent for the same cell type query? If it's not consistent, then it's likely not to do with the backend.

**Review by atarashansky — COMMENTED** [review]
Will finish review tomorrow. Added a few comments for now.

File: frontend/src/views/WheresMyGene/components/CellInfoSideBar/index.tsx [hunk 3]
```diff
@@ -0,0 +1,115 @@
+import { HTMLTable } from "@blueprintjs/core";
```
Comment (tihuan) [inline frontend/src/views/WheresMyGene/components/CellInfoSideBar/index.tsx:1]: Ah can we add a TODO to use SDS Table please? Let's stop using Blueprint for new features, so it's easier to remove Blueprint down the road 👌 Thank you!

**Review by tihuan — COMMENTED** [review]
Awesome work!

**Review by atarashansky — CHANGES_REQUESTED** [review]
Is this shrink in width of the landing screen intended? It's currently not behind the feature flag.

image

**Review by atarashansky — COMMENTED** [review]
Also, I've been running into this error periodically (see below). It typically happens when I'm loading a tissue or a gene. It seems like when the tissues/genes change, `tissuesByName` becomes temporarily undefined or something.

image

**Comment by atarashansky** [comment]
> Also, I've been running into this error periodically (see below). It typically happens when I'm loading a tissue or a gene. It seems like when the tissues/genes change, `tissuesByName` becomes temporarily undefined or something.
> 
> image

Hmmm, might be something funky going on with my local env. I haven't been able to reproduce after restarting my server.

File: frontend/src/views/WheresMyGene/components/CellInfoSideBar/index.tsx (hunk 3 above)
Comment (seve) [reply frontend/src/views/WheresMyGene/components/CellInfoSideBar/index.tsx]: SDS table is too heavy weight for this table, we just needed a lightly styled html table and the blueprint table got us 95% of the way there

**Comment by seve** [comment]
> After clicking `Copy`, the outline stays highlighted until you click somewhere else: image

@tihuan Is this functionality we want to disable? It's A11y right?

File: frontend/src/views/WheresMyGene/components/CellInfoSideBar/index.tsx [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+export interface CellInfoBarProps {
+  cellInfoCellType: Exclude<State["cellInfoCellType"], null>;
+  tissueName: string;
+}
+function CellInfoSideBar({
+  cellInfoCellType,
+  tissueName,
+}: CellInfoBarProps): JSX.Element | null {
+  const urlParams = new URLSearchParams(window.location.search);
+  let testType: "binomtest" | undefined = undefined;
```
[the remark this answers is not in the document]
Comment (seve) [reply frontend/src/views/WheresMyGene/components/CellInfoSideBar/index.tsx:23]: fixed

[the remark this answers is not in the document]
Comment (atarashansky) [reply frontend/src/views/WheresMyGene/components/CellInfoSideBar/index.tsx]: i think you might be missing some commits

File: frontend/src/views/WheresMyGene/components/GetStarted/style.ts [hunk 5]
```diff
@@ -9,14 +9,25 @@ export const Header = styled.h1`
...
-  display: flex;
   position: absolute;
   top: 150px;
   margin-left: -10px;
-  width: calc(96vw - ${EXPANDED_WIDTH_PX + CELL_INFO_SIDEBAR_WIDTH_PX}px);
+
+  display: ${({ isHidden }: WrapperProps) => (isHidden ? "none" : "flex")};
+
+  width: ${({ fmg }: WrapperProps) => {
+    console.log(fmg);
```
Comment (atarashansky) [inline frontend/src/views/WheresMyGene/components/GetStarted/style.ts:26]: remove

**Review by atarashansky — APPROVED** [review]

File: frontend/src/views/WheresMyGene/components/GetStarted/style.ts (hunk 5 above)
Comment (seve) [reply frontend/src/views/WheresMyGene/components/GetStarted/style.ts:26]: ty

**Comment by tihuan** [comment]
> > After clicking `Copy`, the outline stays highlighted until you click somewhere else: image
> 
> @tihuan Is this functionality we want to disable? It's A11y right?

Ah this bug is fixed in my SDS MUIv5 migration branch! I just merged the latest `main` and tested it and it no longer shows border on click. The a11y style should be using `:focus-visible`, which only shows the border when the browser detects keyboard interaction 👌

File: frontend/src/views/WheresMyGene/components/CellInfoSideBar/index.tsx (hunk 3 above)
Comment (tihuan) [reply frontend/src/views/WheresMyGene/components/CellInfoSideBar/index.tsx]: Ah actually SDS Table is only available in SDS with MUIv5, so I just migrated it in my migration PR!

https://github.com/chanzuckerberg/single-cell-data-portal/pull/3734

Screenshot 2022-12-13 at 6 09 21 PM

**Pull request merged**